src/muya/lib/contentState/paragraphCtrl.js

```diff
@@ -68,10 +68,13 @@ const paragraphCtrl = ContentState => {
   ContentState.prototype.handleFrontMatter = function () {
     const firstBlock = this.blocks[0]
     if (firstBlock.type === 'pre' && firstBlock.functionType === 'frontmatter') return
+
+    // Default front matter language is YAML. Others cannot be selected via UI.
```

Comment:
Do we need a setting option that allow user to select the default front matter style `YAML`, `JSON(;;; OR {})`, `TOML`?